src/app/zap-templates/zcl/data-model/chip/resource-monitoring-cluster.xml

```diff
@@ -30,6 +30,7 @@ limitations under the License.
     <attribute side="server" code="0x0001" define="DEGRADATION_DIRECTION" type="DegradationDirectionEnum" min="0" max="1" writable="false" isNullable="false" optional="true">DegradationDirection</attribute>
     <attribute side="server" code="0x0002" define="CHANGE_INDICATION" type="ChangeIndicationEnum" min="0" max="2" writable="false" isNullable="false" default="0" optional="false">ChangeIndication</attribute>
     <attribute side="server" code="0x0003" define="IN_PLACE_INDICATOR" type="boolean" writable="false" isNullable="false" optional="true">InPlaceIndicator</attribute>
+    <attribute side="server" code="0x0004" define="LAST_CHANGED_TIME" type="epoch_s" writable="true" isNullable="true" optional="true">LastChangedTime</attribute>
```

Comment:
Other attributes specify access in their definition. Looking at network monitoring cluster as an example, I imagine access would be like:

```
<attribute side="server" ...>
   <description>...</description>
   <access op="read" privilege="view"/>
   <access op="write" privilege="operate"/>
</attribute>
```

Should this be set here? or any reason for it not be set (trying to understand the access of `VO` in the spec text.

Reply:
So we checked the spec for other examples, we found various occurrences of attributes with `RW VO` (e.g. On / Off cluster, attribute: OnTime) where the XML for this cluster is exactly like we implemented it.

In addition the entire spec has no `privilege="operate"` in any file. 

Could the text of the core spec be the reason for this? 
> An attribute with access 'RW' defaults to access 'RW VO'. A request command with implicit invoke access defaults to privilege 'O'.